docs/tac-skills.md

```diff
@@ -53,20 +55,24 @@ aea create tac_participant_two
 ### Add the tac participation skill to participant one
 ``` bash
 cd tac_participant_one
+aea add connection oef
 aea add skill tac_participation
 aea add skill tac_negotiation
+aea install
 ```
 
 ### Add the tac participation skill to participant two
```

Comment:
in weather skill, the title is:
` ### Add the oef connection and the weather station skill `

here the `oef connection` part is missing. Would be nice to normalize the description of this step across skills tutorials.
Some ideas:
- remove the `add oef connection` part 
- change with `set the X skill up`

Reply:
Yeah we should refactor all guides once we have come to 0.2.0. For now I will leave it.